plugins/Diagnostics/Diagnostic/ReportInformational.php

```diff
@@ -42,8 +42,8 @@ public function execute()
             $results[] = DiagnosticResult::informationalResult('Had visits in last 1 day', $this->hadVisitsInLastDays(1));
             $results[] = DiagnosticResult::informationalResult('Had visits in last 3 days', $this->hadVisitsInLastDays(3));
             $results[] = DiagnosticResult::informationalResult('Had visits in last 5 days', $this->hadVisitsInLastDays(5));
-            $results[] = DiagnosticResult::informationalResult('Archive Time Last Started', Option::get(CronArchive::OPTION_ARCHIVING_STARTED_TS));
-            $results[] = DiagnosticResult::informationalResult('Archive Time Last Finished', Option::get(CronArchive::OPTION_ARCHIVING_FINISHED_TS));
+            $results[] = DiagnosticResult::informationalResult('Archive Time Last Started', date("Y-m-d H:i:s", Option::get(CronArchive::OPTION_ARCHIVING_STARTED_TS)));
+            $results[] = DiagnosticResult::informationalResult('Archive Time Last Finished', date("Y-m-d H:i:s", Option::get(CronArchive::OPTION_ARCHIVING_FINISHED_TS)));
```

Comment:
For a new installation where the archiving might have never been started or finished, the option values might not exist. In that case I guess `1970-01-01 01:00:00` would be displayed. We could maybe simply output a `-` for that case (e.g. if the `Option::get` returns false).

Reply:
Adding a suggestion to make it easier to implement @sgiehl's solution - @mikkeschiren can you please review and commit this change if satisfied? Then we can merge this PR I think.
```suggestion
            $archive_start = Option::get(CronArchive::OPTION_ARCHIVING_STARTED_TS);
            $results[] = DiagnosticResult::informationalResult('Archive Time Last Started', ($archive_start ? date("Y-m-d H:i:s", $archive_start) : '-'));
            
            $archive_end = Option::get(CronArchive::OPTION_ARCHIVING_FINISHED_TS);
            $results[] = DiagnosticResult::informationalResult('Archive Time Last Finished', ($archive_end ? date("Y-m-d H:i:s", $archive_end) : '-'));
```